shell/platform/android/android_shell_holder.h

```diff
@@ -19,17 +20,82 @@
 
 namespace flutter {
 
+//----------------------------------------------------------------------------
+/// @brief      This is the Android owner of the core engine Shell.
+///
+///             This is the top orchestrator class on the C++ side for the
```

Comment:
I think you need to add `@details` to avoid this from all getting jammed into the brief description.

Reply:
+1